### PR TITLE
[Improvement][Seatunnel-web][Hive-JDBC] Add Hive-JDBC into seatunnel-web

### PR DESCRIPTION
**Pull request by arshadmohammad**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs

### CODE REVIEW

**Comment by arshadmohammad** [comment]
Hive-Jdbc functionality verification snapshots
image

image

image

File: seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java [hunk 1]
```diff
@@ -28,17 +28,17 @@
 public class HiveJdbcDataSourceFactory implements DataSourceFactory {
     @Override
     public String factoryIdentifier() {
-        return "Hive-JDBC";
+        return "JDBC-Hive";
```
Comment (wuchunfu) [inline seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java:31]: @arshadmohammad Can `JDBC-Hive` be extracted as a common variable parameter

File: seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java [hunk 2]
```diff
@@ -28,17 +28,17 @@
...
     @Override
     public Set<DataSourcePluginInfo> supportedDataSources() {
         DataSourcePluginInfo dataSourcePluginInfo =
                 DataSourcePluginInfo.builder()
-                        .name("Hive-JDBC")
+                        .name("JDBC-Hive")
                         .type(DatasourcePluginTypeEnum.DATABASE.getCode())
                         .version("1.0.0")
-                        .icon("Hive-JDBC")
+                        .icon("JDBC-Hive")
```
Comment (wuchunfu) [inline seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java:41]: `JDBC-Hive` uses the same common variable parameter

File: seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java (hunk 1 above)
Comment (arshadmohammad) [reply seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java:31]: sure, Thanks @wuchunfu  for the review.

File: seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java (hunk 2 above)
Comment (arshadmohammad) [reply seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java]: The review comment has been addressed

File: seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java (hunk 1 above)
Comment (arshadmohammad) [reply seatunnel-datasource/seatunnel-datasource-plugins/datasource-jdbc-hive/src/main/java/org/apache/seatunnel/datasource/plugin/hive/jdbc/HiveJdbcDataSourceFactory.java]: The review comment has been addressed

**Comment by arshadmohammad** [comment]
ping @Hisoka-X

**Review by Hisoka-X — APPROVED** [review]

**Comment by Hisoka-X** [comment]
cc @wuchunfu

**Comment by Hisoka-X** [comment]
waiting @wuchunfu

**Review by wuchunfu — APPROVED** [review]
LGTM, thanks @arshadmohammad

**Comment by arshadmohammad** [comment]
Thanks @wuchunfu , @Hisoka-X  for reviewing and merging this PR

**Pull request merged**